ui/desktop/app/styles/app.scss

```diff
@@ -173,7 +173,6 @@
   .copyable-button {
     margin: 0;
     padding: 0 sizing.rems(xxs);
-    position: relative;
```

Comment:
@randallmorey I deleted this `position: relative` because was the cause of the copy icon messing up with the `z-index`. I tested the uses cases I am aware and did change nothing, but let me know your thoughts.